Week1/helloInDifLang.js

```diff
@@ -0,0 +1 @@
+console.log ('hello world  ' + '\nздравствуйте мира  ' + '\nmerhaba dunya'); 
```

Comment:
Hey this looks good,
but I think that the assignment was to add a separate `console.log` for each language, so can you change that:)?

Reply:
console.log for each languages. done